notebooks/lesson_preprocessor.py

```diff
@@ -141,11 +190,11 @@ def END_OF_EXERCISE(self, forum_cta=1):
 # Keep Going
 
 When you're ready to continue, [click here]({}) to continue on to the next tutorial on {}.""".format(
-        next.tutorial_url, next.topic,
+        next.tutorial.url, next.topic,
         )
         return res
 
-        # Alternative formulation (used on days 5 and 6):
+        # Alternative formulation (used on days 5 and 6 of Python challenge):
         # Want feedback on your code? To share it with others or ask for help, you'll need to make it public. Save a version of your notebook that shows your current work by hitting the "Commit & Run" button. Once your notebook is finished running, go to the Settings tab in the panel to the left (you may have to expand it by hitting the [<] button next to the "Commit & Run" button) and set the "Visibility" dropdown to "Public".
```

Comment:
You can change visibility even from edit mode now in the right sidebar.  

More broadly, I don't think we need to include all this.  Occasionally (~0.1% of the time) users could use this information. But it's just extra verbage for 99.9% of learn users, who never hit this issue in the absence of the message.

Reply:
Yup, this text is pretty old. It's actually cribbed from Rachael's 5-day Challenge notebooks ([e.g.](https://www.kaggle.com/rtatman/data-cleaning-challenge-handling-missing-values)). 

I'm open to changing this, but I'd want to give it a little thought before removing it. It is nice when users posting on the forums have already made their kernel public and you can skip that extra roundtrip. Maybe the right solution is something like a sticky "read this first" thread at the top of the Learn Forum that has instructions about sharing your kernel publicly.

Reply:
**It is nice when users posting on the forums have already made their kernel public and you can skip that extra roundtrip.**

Sure.  But this has probably come up on the order of 20 times... and we've had >200k forks. That's ~0.01%

Stickied post at the top of the forum seems reasonable